graphene_sqlalchemy/tests/test_converter.py

```diff
@@ -270,23 +270,19 @@ def test_should_postgresql_uuid_convert():
 
 
 def test_should_postgresql_enum_convert():
```

Comment:
nit. It may make sense to name this `test_should_postgresql_enum_convert_enum` for consistency. Same for `test_should_postgresql_py_enum_convert`